test/src/e2e_vm_tests/test_programs/should_pass/stdlib/ge_test/Forc.toml

```
@@ -0,0 +1,9 @@
+[project]
+authors = ["Alexander Hansen"]
+entry = "main.sw"
+license = "Apache-2.0"
+name = "ge_test"
+
+[dependencies]
+core = { path = "../../../../../../../sway-lib-core" }
+std = { path = "../../../../../../../sway-lib-std" }
```

And update lockfile too
```suggestion
std = { path = "../../../../../../../sway-lib-std" }
```